ci/scripts/github/build_wheel.sh

```
@@ -76,6 +90,10 @@ for whl in "${MOVED_WHEELS[@]}"; do
         UV_PIP_OUT=$(uv pip install -q --prerelease=allow --find-links "${WHEELS_BASE_DIR}" "${whl}" 2>&1)
         INSTALL_RESULT=$?
 
+        # Report the packages in the environment regardless of install success
+        rapids-logger "Installed wheel ${whl} with Python ${pyver}, pip install exit code ${INSTALL_RESULT}"
+        uv pip list --format json > ${PIP_REPORTS_DIR}/$(basename "${whl}" .whl)_py${pyver}_packages.json
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)